src/NServiceBus.TransportTests/When_processing_fails.cs

```diff
@@ -0,0 +1,41 @@
+namespace NServiceBus.TransportTests
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Threading.Tasks;
+    using NUnit.Framework;
+    using Transport;
+
+    public class When_processing_fails : NServiceBusTransportTest
```

Comment:
Once we add OnComplete this test will be modified to make sure that the context is floated there as well